### DISCUSSION TITLE
Double Press

### DISCUSSION THREAD

**Discussion by Solido** [btwld/mix#447] [Q&A]
I need support for double click :) 
right now onPress only is supported if I understand well?
Thanks!

**Comment by leoafarias** [comment]
@Solido, do you need just the callback on Pressable, or do you need the utility to be able to style it?

**Reply by Solido** [reply]
I'm working on a multi platforms app so I would just require the event to conform.
In the future, desktop only devs would certainly require the styling but it's marginal I guess.
Should press and double press trigger the same style?
Thanks Leo!
